
import "proto/pbSpell.proto";

message pbCreateClient
{
    required int32 hp = 1;
    required int32 mp = 2;
    repeated int32 props = 3;
    repeated pbSpellClient spells = 4;
}
message pbCreateDB
{
    required int32 hp = 1;
    repeated int32 props = 2;
    required int32 level = 3;
}
